_includes/markdown/Systems.md

```diff
@@ -252,3 +253,94 @@ Once tuning moves beyond these main items, performance improvements will be mino
+
+### Memcached
+
+Memcached is simpler than Redis and has fewer features. While Redis can be used as a full database, memcached is only a key-value store. This simplicity is by design and makes Memcached a very low maintenance tool. 
+
+#### High Availability
+
+Memcached can be installed on multiple servers that can combine into a memcached “pool”.  Keeping with the theme of simplicity, each memcached server in the pool knows nothing about the other instances in the pool and operates completely independently.  It is up to the client (in this case, WordPress and PHP) to distribute data across the pool of memcached servers however it sees fit.  Memcached performs no replication, failover, or connection balancing itself.  To distribute data evenly across multiple servers, the PHP extensions for memcached use a hashing strategy.  In this way, multiple PHP web servers can read and write to multiple memcached servers and all know exactly where each key is stored.  If one server fails, keys are distributed across the remaining memcached instances.  This can have a hit on performance (especially compared to replication scheme) as keys need to be regenerated to replace the missing data, but is by far the easiest type of distributed cache to setup.
+
+#### Item Size
+
+By default, memcached accepts items (key + value) of 1 MB or less. In most scenarios, this is fine and plenty of space. However, on larger WordPress sites or sites with a number of plugins, the “alloptions” array combining all autoloaded rows from the wp_options table can exceed 1 MB.  This array will be stored in memcached if memcached is in use for the object cache, unless it exceeds the memcached item size limit.  When this array is larger than the memcached item size, it can cause all sorts of odd issues and inconsistent performance.  
```

Comment:
This is only one example. We should point out that this might also apply for other scenarios like caching remote calls, query results or html fragments.

Reply:
I'll address this in the next commit.